libs.ts

```diff
@@ -1382,8 +1454,46 @@ export const libs: Lib[] = [
 		},
 		userApps: 'No',
 		polls: 'Yes',
-		forwarding: 'No',
-		appEmoji: 'Yes'
+		forwarding: 'Yes',
+		appEmoji: 'Yes',
+		componentsV2: {
+			text: 'Has a PR',
+			url: 'https://github.com/discord-php/DiscordPHP/pull/1294'
+		}
+	},
+	{
+		name: 'laracord',
+		url: 'https://github.com/laracord/framework',
+		language: 'PHP',
+		apiVer: 10,
+		gwVer: 10,
+		voiceVer: 1,
+		slashCommands: 'Yes',
+		buttons: 'Yes',
+		selectMenus: 'Yes',
+		threads: 'Yes',
+		guildStickers: 'Yes',
+		contextMenus: 'Yes',
+		autocomplete: 'Yes',
+		scheduledEvents: 'Yes',
+		timeouts: 'Yes',
+		modals: 'Yes',
+		permsv2: 'Yes',
+		automod: 'Yes',
+		localization: 'Yes',
+		forums: 'Yes',
+		monetization: {
+			text: 'Has a dependant PR',
+			url: 'https://github.com/discord-php/DiscordPHP/pull/1186'
+		},
+		userApps: 'No',
+		polls: 'Yes',
+		forwarding: 'Yes',
+		appEmoji: 'Yes',
+		componentsV2: {
+			text: 'Has a PR',
+			url: 'https://github.com/laracord/framework/pull/134'
+		}
 	},
```

Comment:
can this be moved to a separate pr, so this pr doesnt add any new libraries

Reply:
I'll let the lib owner know